fedora-messaging.spec

```diff
@@ -0,0 +1,132 @@
+%{!?__python2: %global __python2 %__python}
+%{!?python2_sitelib: %global python2_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")}
+
+%global pkgname fedora-messaging
+%global srcname fedora_messaging
+%global desc \
+Tools and APIs to make working with AMQP in Fedora easier.
```

Comment:
IMO, spec files aren't that useful to have upstream since they are very distro specific, and it's painful to keep upstream and downstream synchronized. Bodhi used to have one in its GitHub repo and I removed it in favor of maintaining the one in dist-git.

Reply:
Yeah I agree, I'm putting it here for review mainly. Then I'll probably remove it.

Reply:
I agree with @bowlofeggs. If you want to just take this specfile and submit it as a package to Fedora I'm happy to review it now. We can build it in rawhide only until we release 1.0.0, which will be long before f30 branches.